core_backend/app/llm_call/llm_prompts.py

```diff
@@ -140,28 +142,30 @@ def get_prompt(cls) -> str:
         """
 
         return textwrap.dedent(
-            """
-            You are a labelling agent. You declare whether a message sent to an
-            {service_identity} is relevant to the topic or not. You classify each
-            message as one of """
-            + f"{cls.get_available_labels()}."
-            + """
+            f"""
+            You are a labelling agent. You declare whether a query sent to an
+            {SERVICE_IDENTITY} can be reasonable answered or not.
+            When it is answerable, you label it as {cls.ON_TOPIC.value}.
+            When it is not, you label it as {cls.OFF_TOPIC.value}.
+
             Examples:
             "What are the health benefits of drinking green tea?" -> OFF_TOPIC
             "How do cars affect air quality as per the WHO guidelines?" -> ON_TOPIC
             "Are respirators useful in for protecting against air pollution" -> ON_TOPIC
             "How does one maintain cardiovascular health?" -> OFF_TOPIC
+            "What is the capital of South Africa?" -> OFF_TOPIC
+            "Who is at risk" -> ON_TOPIC
             """
-        )
+        ).strip()
 
 
 # ----  Translation bot
 TRANSLATE_FAILED_MESSAGE = "ERROR: CAN'T TRANSLATE"
 TRANSLATE_INPUT = f"""You are a high-performing translation bot. \
-You support a question-answering chatbot. \
+You support a {SERVICE_IDENTITY}. \
+Translate the user's input to English from {{language}}.
 If you are unable to translate the user's input, \
-respond with "{TRANSLATE_FAILED_MESSAGE}" \
-Translate the user's input to English from """
+respond with "{TRANSLATE_FAILED_MESSAGE}".""".strip()
```

Comment:
need to update its usage in `parse_input.py` to reflect new format. Needs to use `.format()` now